config/config.go

```diff
@@ -1035,3 +1035,14 @@ func (p *PackageConfig) sanitizedVersion() string {
 	// replaces all the . if they exist with _
 	return strings.ReplaceAll(p.Version, ".", "_")
 }
+
+// NewConfigValidationError returns a config validation error occurring at a given path.
+func NewConfigValidationError(path string, err error) error {
```

Comment:
These were pulled from goutils to drop the `github.com/pkg/errors` package. Unfortunately there was no way to control how those errors get outputed with zap's json encoder.